test/unit/test_prune.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
this should be 2018 for new files. But why adding a new file? `test_inference.py` would also work.